tfhe/src/boolean/engine/mod.rs

```diff
@@ -257,6 +260,41 @@ impl Default for BooleanEngine {
 }
 
 impl BooleanEngine {
+    /// Replace the thread_local BooleanEngine
+    ///
+    /// `new_engine` will replace the already_existing
+    /// `thread_local` engine.
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// use tfhe::boolean::engine::BooleanEngine;
+    /// use tfhe::core_crypto::commons::generators::DeterministicSeeder;
+    /// use tfhe::core_crypto::commons::math::random::Seed;
+    /// use tfhe::core_crypto::prelude::ActivatedRandomGenerator;
+    ///
+    /// // WARNING: Using a deterministic seed is not recommanded
+    /// // as renders the random generation insecure
+    ///
+    /// let deterministic_seed = Seed(0);
+    ///
+    /// // We change the engine in the main thread
+    /// // then generate a client key, and then encrypt
+    /// // a boolean value and serialize it to compare
+    /// // it with other ciphertext
```

Comment:
looks like this comment is not accurate

Reply:
Ah yes it good'ol copy paste from the test